packages/ses/package.json

```diff
@@ -48,7 +48,7 @@
     "build": "node scripts/bundle.js",
     "clean": "rm -rf dist",
     "cover": "c8 ava",
-    "demo": "http-server -o /demos",
+    "demo": "python3 -m http.server",
```

Comment:
This is unfortunate, we shouldn't force a python dependency just for a static server. There ought to be a simple enough node based static http server! Doesn't [serve](https://www.npmjs.com/package/serve) fit the bill?